test/parallel/test-handle-wrap-close-abort.js

```diff
@@ -1,16 +1,19 @@
 'use strict';
-require('../common');
+const common = require('../common');
 
-process.on('uncaughtException', function() { });
+function noop() {}
```

Comment:
No good reason. Aesthetics. Or indecisiveness. Or something. If you prefer it all inline, I'm happy to do that instead.